oauthenticator/globus.py

```diff
@@ -79,7 +96,7 @@ def _authorize_url_default(self):
     ).tag(config=True)
 
     def _identity_provider_default(self):
-        return os.getenv('IDENTITY_PROVIDER', 'globusid.org')
+        return os.getenv('IDENTITY_PROVIDER', '')
```

Comment:
This is another change unrelated to streamlining dependencies, but I think it's preferable. The previous value of `globusid.org` essentially locks down the default installation to an identity almost nobody uses. Setting this to the empty string allows the user to login with any IdP initially.